jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -65,24 +66,46 @@ public interface Response extends Content.Sink
 
     CompletableFuture<Void> writeInterim(int status, HttpFields headers);
 
-    // TODO: make it static, otherwise we must override it in Wrapper.
-    default boolean writeTrailers(Content.Chunk chunk, Callback ignored)
+    /**
+     * <p>Return a representation of the response as a chunk handler suitable for being
+     * passed to the {@link Content#copy(Content.Source, Content.Sink, BiPredicate, Callback)}
+     * method.  Specifically, as a chunk handler that will handle {@link Trailers} chunks
+     * by adding the chunk fields to the {@link HttpFields} supplied by
+     * {@link Response#getTrailersSupplier()}.</p>
+     * <p>This is specifically useful for writing chunks that have been received via
+     * the {@link Content.Source#read()} API, for example when echoing a request to a response:</p>
+     * <pre>
+     *   Content.copy(request, response, Response.asTrailerChunkHandler(response), callback);
+     * </pre>
+     * @param response The response for which to process trailer chunks.
+     * @return A chunk handler that will add trailer chunks to the response's trailer supplied fields.
+     * @see Content#copy(Content.Source, Content.Sink, BiPredicate, Callback)
+     * @see Trailers
+     */
+    static BiPredicate<Content.Chunk, Callback> asTrailerChunkHandler(Response response)
```

Comment:
Bit of a mouthful. Can't we just keep it named `writeTrailers`?
Or perhaps `trailersWriter` to indicate that it returns a writer that will write the trailers?

Reply:
I think the new name is good:
 + It does not write trailers, it copies them from one HttpFields to another, to be written at a later time.
 + If it was just writing/setting trailers, then the signature would be a lot different.  It has a very specific signature so that it can be used for a very specific purpose: to be a chunk handler for a copy operation.
 + It could perhaps just be `asChunkHandler`, so that in future other types of chunk might be handled, but I doubt we will ever do that.  Having a name that tells us it is used for trailers will help coders looking for a way to handle `Trailers` chunks

Reply:
`Handler` is so overloaded...

Taking into account your points, how about `Response.trailersCopier(Response)`?

The "as" in front should be dropped as it is not "converting" the response to something else -- I would tolerate "get" as in `getTrailersCopier`, but then it creates a new one every time, so `newTrailersCopier`, but then dropping the prefix make it nicer:

`Content.copy(request, response, Response.trailersCopier(response), callback);`

Reply:
But the argument in the copy signature is called `chunkHandler`, so the names have to match (and that is your name choice in Content!).
I could rename the arg to `chunkTransformer`? but then this isn't transforming it?  or `chunkProcessor`, but then `Processor` is overloaded as well.

But regardless,  this PR is just about fixing a TODO in the `Response` class.   If you want to change the name of the `chunkHandler` in the `Content` class, then do that in a different PR?